ui/src/testing/factories/state.ts

```diff
@@ -6,24 +6,32 @@ import { user } from "./user";
 import type { AuthState, UserState } from "app/store/user/types";
 import type { MessageState } from "app/store/message/types";
 import type { NotificationState } from "app/store/notification/types";
+import type { PodState } from "app/store/pod/types";
 
-export const authState = define<AuthState>({
-  errors: null,
-  loaded: true,
+const defaultState = {
+  errors: () => ({}),
```

Comment:
This might be OK, but currently not all models have errors that are an object. We have a card about the API being inconsistent.